lib/smooth/include/smooth/core/filesystem/MMCSDCard.h

```diff
@@ -18,6 +18,14 @@ limitations under the License.
 #pragma once
 
 #include "SDCard.h"
+
+#pragma GCC error "Your chip does not support SDMMC_HOST. See https://github.com/PerMalmberg/Smooth/issues/171"
+
+#pragma message("Target system is:" CONFIG_IDF_TARGET)
+#pragma message(                                                                                       \
```

Comment:
@PerMalmberg take a look at this version. The best I've managed to achieve so far.

Reply:
This will do nicely!